test/unit/models/embassy_test.rb

```diff
@@ -152,4 +152,34 @@ class EmbassyTest < ActiveSupport::TestCase
       assert_equal expected, embassy.remote_office
     end
   end
+
+  context "when there is an organisation with two offices in two remote countries and one isn't an embassy" do
+    before do
+      organisation = create(:worldwide_organisation,
+                            world_locations: [world_location],
+                            name: "org-name",
+                            slug: "org-slug")
+      @second_location = create(:world_location)
+      @third_location = create(:world_location)
+      second_location_contact = create(:contact, street_address: "street-address", country: @second_location)
```

Comment:
Minor: Do you need to specify the `street_address` here? It doesn't seem to be used in the test and it looks as if the factory will set one by default.

Reply:
There's a contact_with_country factory that we can use, I've made that change in cfba5a4ed7